src/main/java/frc/robot/subsystems/DriveTrain.java

```diff
@@ -64,26 +63,39 @@
 
     public DriveTrain() {
         // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTRUCTORS
-        leftFollower = new WPI_TalonFX(0);
+leftFollower = new WPI_TalonFX(0);
+
+
+        
+rightFollower = new WPI_TalonFX(2);
+
 
-        rightFollower = new WPI_TalonFX(2);
+        
+shifter = new DoubleSolenoid(10, 2, 4);
+addChild("shifter",shifter);
 
-        shifter = new DoubleSolenoid(10, 2, 4);
-        addChild("shifter", shifter);
+        
+pigeon = new PigeonIMU(4);
 
-        pigeon = new PigeonIMU(4);
 
-        rightMaster = new WPI_TalonFX(3);
+        
+rightMaster = new WPI_TalonFX(3);
 
-        leftMaster = new WPI_TalonFX(1);
 
-        tankDrive = new DifferentialDrive(rightMaster, leftMaster);
-        addChild("TankDrive", tankDrive);
-        tankDrive.setSafetyEnabled(false);
-        tankDrive.setExpiration(0.1);
-        tankDrive.setMaxOutput(1.0);
+        
+leftMaster = new WPI_TalonFX(1);
 
-        // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTRUCTORS
+
+        
+tankDrive = new DifferentialDrive(rightMaster, leftMaster);
+addChild("TankDrive",tankDrive);
+tankDrive.setSafetyEnabled(false);
+tankDrive.setExpiration(0.1);
+tankDrive.setMaxOutput(1.0);
+
+        
+
+    // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTRUCTORS
         motorConfig();
```

Comment:
think we want motorconfig in teleop & period init to start & then here -- comment this out for now?